src/scss/_grid-areas.scss

```diff
@@ -156,6 +156,7 @@
 				margin-right: -1em;
 
 				&[aria-current="location"] {
+					color: oColorsGetPaletteColor('teal-30');
```

Comment:
Is this possible to test with pa11y?

Reply:
Yep, and we should have a Pa11y demo for this project :) going to do that separately so I don't lose my flow with the new site

Reply:
#19 opened so I don't forget